README.md

```diff
@@ -81,15 +81,16 @@ mkosi binary and is not to be considered a public API.
 
 ## kernel-install plugin
```

Comment:
plugin => plugins

Reply:
Actually I would document these separately and as a "one or the other" type of setup. We shouldn't recommend users use both. Either they use mkosi-initrd to build their own UKI with local modifications or they use a vendor provided UKI and mkosi-addon for local modifications. But it should be one or the other, not both at once.